tests/test_schema_validation.py

```diff
@@ -5,41 +5,96 @@
 from octopoes.ddl.ddl import SchemaLoader, SchemaValidationException
 
 
+def schemavalidation(schema: str, expected_output: str) -> bool:
```

Comment:
Please see also pytest's `parametrize` [decorator](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions)

If you can gather a list of test input and expected output it would make this test much smaller and easier to extend.

Reply:
Done!